pkg/kubepkg/kubepkg.go

```diff
@@ -101,6 +101,18 @@ var (
 	}
 )
 
+type KubePKG struct {
```

Comment:
I've consistently had trouble with naming things in this package :sweat_smile: 
Can we try maybe `KubepkgClient`?

Open to other suggestions as well, but we should capitalize `PKG` (it's not an acronym for anything).

Reply:
Hm, how about only `Client`? We already have the kubepkg as package name so people can recognize it via `kubepkg.Client` and construct via `kubepkg.New()`. Should be fine, right?

Reply:
@saschagrunert -- Works for me and removes the stutter :)

Reply:
Awesome, good :)